doc/tutorials/gpu-passthru.rst

```diff
@@ -61,6 +61,15 @@ The example extends the information provided in the :ref:`gsg`.
 #. In the ACRN Configurator, create a shared scenario with a Service VM and one
    post-launched User VM.
 
+#. Select the Service VM and go to **Basic Parameters > Kernel
+   command-line parameters**.
+
+#. Append ``i915.modeset=0`` in **Kernel command-line parameters**.
```

Comment:
Can we add a (short) explanation as to what that does and why we need it?

We could even add this one to this doc ;-) https://projectacrn.github.io/latest/user-guides/kernel-parameters.html

Reply:
updated.